src/Zynga/Framework/Cache/V2/Driver/Memcache.hh

```diff
@@ -125,9 +162,14 @@ class Memcache extends DriverBase {
       }
 
       // add the host / port combinations to the memcache object, addserver
-      //   always returns true as it lazy connects at use time.
+      // always returns true as it lazy connects at use time.
       foreach ($serverPairs as $host => $port) {
-        $memcache->addserver($host, $port);
+
+        // addserver does not check for duplicates
+        if ($this->_registeredHosts->containsKey($host) === false) {
```

Comment:
FYI - i don't think this matters with Memcache. I think you are confusing 'Memcache' vs 'Memcached' (notice the 'd' at the end) in php. This is something that has confused me in the past as well. No harm in adding this check though.

Reply:
Great to know thanks!